alpharaw/mzml.py

```diff
@@ -10,10 +10,38 @@
 
 
 class MzMLReader(MSData_Base):
+    """
+    Load mzml file as `:class:`MSData_Base` structure.
+    Register "mzml" in :data:`ms_reader_provider`.
+
+    Parameters
+    ----------
+    filename : str
+        mzml file path.
+
+    Returns
+    -------
+    dict
+        Spectrum information dict.
+    """
+
     def _import(
         self,
         filename: str,
-    ):
+    ) -> dict:
+        """
+        Re-implementation of :func:`alpharaw.MSData_Base._import`.
```

Comment:
why Re-implementation? isn't this a implementation specific to the mzml format? also, maybe add smth like "See documentation of superclass for details."

Reply:
_import is a interface. `Implementation` would be better